examples/basic-checkpoint-ha.yaml

```diff
@@ -25,6 +25,7 @@ spec:
   image: flink:1.14.3
   flinkVersion: 1.14.3
   flinkConfiguration:
+    taskmanager.numberOfTaskSlots: "2"
```

Comment:
@gyfora as follow-up I would suggest that we can handle yaml number types and don't assume Configuration.setString will just work. That way users won't need to quote numbers..